pandas/tests/indexing/multiindex/test_sorted.py

```diff
@@ -90,3 +91,23 @@ def test_series_getitem_not_sorted(self):
         expected.index = expected.index.droplevel(0)
         tm.assert_series_equal(result, expected)
         tm.assert_series_equal(result2, expected)
+
```

Comment:
these should be here: pandas/tests/indexes/multi/test_sorting.py:

Reply:
I've moved them across.